### PR TITLE
Ra 10 arm joint setup

### PR DESCRIPTION
**Pull request by sajidanower23**

### CODE REVIEW

**Comment by sajidanower23** [comment]
Indentation has been fixed.

File: rover/src/owr_drive_controls/CMakeLists.txt [hunk 1]
```diff
@@ -117,6 +117,10 @@ add_executable(cmd_vel_2_joints src/cmdVelToJoints.cpp src/Drive.cpp src/crabDri
 add_dependencies(cmd_vel_2_joints owr_messages)
 target_link_libraries(cmd_vel_2_joints ${catkin_LIBRARIES})
 
+add_executable(cmd_vel_2_arm_joints src/cmdVelToArmJoints.cpp src/armControl.cpp)
+add_dependencies(cmd_vel_2_arm_joints owr_messages)
+target_link_libraries(cmd_vel_2_arm_joints ${catkin_LIBRARIES})
```
Comment (hjed) [inline rover/src/owr_drive_controls/CMakeLists.txt]: Its not really `cmd_vel` that might be a bit confusing as thats the standard name of the ros velocity vector (fot driving)

File: rover/src/owr_drive_controls/include/armControl.hpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*
+ * Original Author: Elliott Smith
+ * Editors:
+ * Date Started: 2/6/18
+ * Purpose: Convert arm joystick message into arm joint control
+ *
```
Comment (hjed) [inline rover/src/owr_drive_controls/include/armControl.hpp]: can we add license comments to these headers please

File: rover/src/owr_drive_controls/include/cmdVelToArmJoints.hpp [hunk 3]
```diff
@@ -0,0 +1,38 @@
+/*
+ * Converts CMD_VEL vectors into arm joint positions
+ * 
+ * Original Author: Elliott Smiht
+ * Editors:
+ * Date Started: 2/6/18
+ */
```
Comment (hjed) [inline rover/src/owr_drive_controls/include/cmdVelToArmJoints.hpp]: ditto

File: rover/src/owr_drive_controls/src/armControl.cpp [hunk 4]
```diff
@@ -0,0 +1,41 @@
+/*
+ * Original Author: Elliott Smith
+ * Editors:
+ * Date Started: 2/6/18
+ * Purpose: Convert arm joystick message into arm joint control
+ *
```
Comment (hjed) [inline rover/src/owr_drive_controls/src/armControl.cpp]: license comment

File: rover/src/owr_drive_controls/src/cmdVelToArmJoints.cpp [hunk 5]
```diff
@@ -0,0 +1,77 @@
+/*
+ * Converts CMD_VEL vectors into arm joint positions
+ * Original Author: Elliott Smith
+ * Editors:
+ * Date Started: 2/6/18
+ */
```
Comment (hjed) [inline rover/src/owr_drive_controls/src/cmdVelToArmJoints.cpp]: license comment

File: rover/src/owr_drive_controls/src/cmdVelToArmJoints.cpp [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+#include <std_msgs/Float64.h>
+#include <geometry_msgs/Vector3.h>
+
+static ArmControl armControl;
+
+int main(int argc, char ** argv) {
+    ros::init(argc, argv, "owr_cmd_vel_2_arm_joints");
+
+    //clawGripAngle = PI/4.0;
```
Comment (hjed) [inline rover/src/owr_drive_controls/src/cmdVelToArmJoints.cpp]: commented code

File: rover/src/owr_drive_controls/src/cmdVelToArmJoints.cpp [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    ros::Rate r(25);
+    while(ros::ok()) {
+        // Publish to all the publishers
+        std_msgs::Float64 msg;
+        msg.data = armUpperActuator;
+        armUpper.publish(msg);
+        r.sleep();
+        msg.data = armLowerActuator;
+      	armLower.publish(msg);
+        r.sleep();
```
Comment (hjed) [inline rover/src/owr_drive_controls/src/cmdVelToArmJoints.cpp]: indentation in this function (looks like tabs vs spaces)

File: rover/src/owr_drive_controls/CMakeLists.txt (hunk 1 above)
Comment (sajidanower23) [reply rover/src/owr_drive_controls/CMakeLists.txt]: Renaming it to `arm_joints`

**Pull request merged**